### PR TITLE
Fix CVE-2019-16377 and clean up Consul::Controller

### PR DESCRIPTION
**Pull request by triskweline**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,45 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
...
+### Compatible changes
+
+- The RSpec matcher `check_power` now also sees powers inherited by a parent controller.
+
+
+## 1.0.2 - 2019-05-22
+
+### Compatible changes
+
+- The `#arity` of of power methods with optional arguments are now preserved.
```
Comment (kratob) [inline CHANGELOG.md]:
```suggestion
- The `#arity` of power methods with optional arguments is now preserved.
```

File: lib/consul/controller.rb [hunk 2]
```diff
@@ -105,18 +68,26 @@ def power(*args)
...
+        unless @consul_power_args_initialized
+          if superclass && superclass.respond_to?(:consul_power_args, true)
+            @consul_power_args = superclass.send(:consul_power_args).dup
+          else
+            @consul_power_args = []
+          end
+          @consul_power_args_initialized = true
+        end
+        @consul_power_args
+      end
```
Comment (kratob) [inline lib/consul/controller.rb]: Isn't this exactly what Active Support's `class_attribute` does?

Comment (triskweline) [reply lib/consul/controller.rb]: I think `class_attribute` only copies values when you write to the attribute. In this case no one ever calls `consul_power_args=`, so the value would never be copied and changes in a child class would affect the parent class.

**Comment by RKushnir** [comment]
Hi, could you, please, release the 1.0.3? bundle-audit fails my builds, asking to upgrade the gem, but there's no new version.

**Pull request merged**